BestBuyCRUD/Employees.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BestBuyCRUD
+{
+    class Employees
```

Comment:
This class will be used to model an individual employee with their own name, email, phone number, etc.  It should probably be named in the singular, e.g. "Employee"